### PR TITLE
add new linux question(s)

### PR DESCRIPTION
**Pull request by Tryxns**
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1613,6 +1613,28 @@ const linuxQuiz = [
     Explanation:
       "Some commands fail to provide enough information and context. When you are in need of a custom solution, using an alias to refer to a more useful base command can be very helpful.",
     Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/#the-linux-alias-command"
+  },
+  {
+    Question:
+      "What does Ctrl + r do in default ubuntu terminal?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1619]:
```suggestion
      "What does Ctrl + r (windows) or ⌘ + r(mac) do in the default ubuntu terminal?",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -1613,6 +1613,28 @@ const linuxQuiz = [
...
+    Answer: "query the command history",
+    Distractor1: "to call find & replace text function",
+    Distractor2: "to redo last action",
+    Distractor3: "to reload current shell",
+    Explanation:
+      "Ctrl + r shortcut helps you query the command history by suggesting autocomplete",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1625]:
```suggestion
      "The Ctrl + r (windows) or ⌘ + r(mac) shortcut helps you query the command history by suggesting autocomplete.",
```

File: src/data/linux-quiz.ts [hunk 3]
```diff
@@ -1613,6 +1613,28 @@ const linuxQuiz = [
...
+    Link: "https://www.freecodecamp.org/news/bash-command-line-tips-to-help-you-work-faster/"
+  },
+  {
+    Question:
+      "In Ubuntu, how to add user named `quincy` to group named `writers`?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1630]:
```suggestion
      "In Ubuntu, which of the following commands adds the user named 'quincy' to a group named 'writers'?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by Tryxns — COMMENTED** [review]
Hello Jessica, thanks for the review, I just resolved all the requests, please re-review..

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**